src/lib/components/FornaContainer.react.js

```diff
@@ -18,6 +18,9 @@ export default class FornaContainer extends Component {
         this._fornaContainer = new PreFornaContainer('#' + id, {
             initialSize: [width, height],
         });
+    }
+
+    componentDidUpdate() {
         this.renderNewSequences();
```

Comment:
I believe you need `renderNewSequences` in BOTH `componentDidUpdate` and `componentDidMount`, as initial render doesn't trigger `componentDidUpdate`

Reply:
Yep you're right! Fixed in 9411e5d.